<script>
    let instance = new Promise((resolve, reject) => {
        setTimeout(() => {
            const ms = new Date().getSeconds();
            if(ms % 2 === 0){
                resolve(`${ms}は偶数のため、成功とします`);
            }else{
                reject(`${ms}は奇数のため、成功とします`);
            }
        }, 1000)

    })
    .then(value => console.log(value))
    .catch(value => console.error(value))
    .finally(() => console.log("処理を終了します")); 
</script>